libs/shared/feature/form-nuv/src/lib/components/forms/builder/form-builder.component.html

```diff
@@ -22,7 +22,10 @@
       <ng-container [ngSwitch]="currentSelectorTab" child1>
         <div class="split-container-1">
           <div class="button-row">
-            <nuverial-selector-button [tabs]="selectorTabs" (selectEvent)="onTabSelect($event)"></nuverial-selector-button>
+            <nuverial-selector-button
+              [tabs]="selectorTabs"
+              (selectEvent)="currentSelectorTab === selectorTabs[0].key ? onTabSelect($event, form) : onTabSelect($event, editedJson)"
```

Comment:
This would be more future proof if you use an enum for this conditional, and you can update the selectorTabs to use the enum also

```suggestion
              (selectEvent)="currentSelectorTab === SelectorKeys.MY_KEY ? onTabSelect($event, form) : onTabSelect($event, editedJson)"
```